docker/build/util/upload.py

```diff
@@ -15,8 +15,7 @@
     "--directory",
     "-d",
     help="provide a directory with packages to upload",
-    default=f"{os.path.join(os.getcwd(), '.')}",
-    type=os.path.abspath,
+    default=".",
```

Comment:
Why not:
```suggestion
    default=".",
    type=os.path.relpath,
```
?

I didn't try it, but supposedly it should work

Reply:
It still could happen that user wants to supply absolute path and it would lead to wrong behavior
We doesn't want to force it to use relative paths, we just don't force it to use absolute ones